subgit/cli.py

```diff
@@ -153,6 +166,12 @@ def run(cli_args, sub_args):
     if cli_args["<command>"] == "status":
         retcode = core.repo_status()
 
+    if cli_args["<command>"] == "delete":
+        repos = sub_args["<repo>"]
+        repos = repos or None
+
+        retcode = core.delete_local_repo(repos)
```

Comment:
Technically we do not differentiate at this point in time between local and remote repo? We have no plans on adding in some command that deletes something on `remote`. We do not separate on this in the commands above either. Give it some thought if `local` should be removed or not from the name